apps/stats-dapp/src/utils/getChipColorByProposalType.ts

```diff
@@ -0,0 +1,26 @@
+import { ChipProps } from '@webb-tools/webb-ui-components/components/Chip/types';
+import { ProposalBatchStatus } from '../generated/graphql';
+
+export function getChipColorByProposalType(
+  type: ProposalBatchStatus
+): ChipProps['color'] {
+  switch (type) {
+    case ProposalBatchStatus.Signed: {
+      return 'green';
+    }
+
+    case ProposalBatchStatus.Removed: {
+      return 'red';
+    }
+
+    case ProposalBatchStatus.Expired: {
+      return 'yellow';
+    }
+
+    default: {
+      throw new Error(
+        'Unknow ProposalStatus inside `getChipColorByProposalType` functioni'
+      );
```

Comment:
Instead of throwing an error, use a default color here to prevent potential interruptions to the UI.